zingolib/src/wallet/transaction_record.rs

```diff
@@ -493,13 +492,13 @@ pub mod mocks {
         /// Sets the output indexes of all contained notes
         pub fn set_output_indexes(mut self) -> Self {
             for (i, toutput) in self.transparent_outputs.iter_mut().enumerate() {
-                toutput.output_index = i as u64;
+                toutput.output_index = Some(i as u64);
```

Comment:
I am guessing from this that u64 is actually correct for transparent output indices?

Reply:
its the code